pkg/Microsoft.ML.AutoMLFeaturizers/Microsoft.ML.AutoMLFeaturizers.nupkgproj

```diff
@@ -0,0 +1,16 @@
+<Project Sdk="Microsoft.NET.Sdk" DefaultTargets="Pack">
```

Comment:
I wonder if `AutoMLFeaturizers` is the best name for this package. Technically, anyone can use these featurizers without using AutoML.

What is going to be the name of the underlying NuGet package that contains the native assemblies? We should probably align with that name. So if that NuGet package is going to be named "AutoMLFeaturizers", then I guess that would be a good name here as well.

Reply:
I will sync with my team today and figure that out. Part of the reason I picked this name is that these Transformers are added specifically for the AutoML team. So while anyone can use them, they are custom tailored for their use.

Reply:
How about naming it NativeFeaturizers? Hinting that implementation is in native code

---
In reply to: [323317152](https://github.com/dotnet/machinelearning/pull/4157#discussion_r323317152) [](ancestors = 323317152)